code/modules/surgery/organs/brain.dm

```diff
@@ -134,7 +134,8 @@
 	icon = 'icons/mob/slimes.dmi'
 	icon_state = "green slime extract"
 	mmi_icon_state = "slime_mmi"
-//	parent_organ = "chest" Hello I am from the ministry of rubber forehead aliens how are you
+	parent_organ = "chest" // Hello I am from the ministry of rubber forehead aliens how are you
+	organ_datums = list(/datum/organ/heart, /datum/organ/lungs)
 
```

Comment:
This is why I chose the chest in the first place, there was an old funny comment with it. I will undo it if needed.